### PR TITLE
Assignment 2a

### PR DESCRIPTION
**Pull request by NataliaBagnoli**

### CODE REVIEW

**Comment by dgarijo** [comment]
@ocorcho

**Comment by ocorcho** [comment]
Assignments 2 and 3 accepted.
Now @dgarijo can check assignment 4

File: Assignment4/NataliaBagnoli-AY3691355/task06.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+from rdflib import FOAF
+
+g.add((ns.Researcher, RDFS.subClassOf, ns.Person))
+
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.3: Create a new individual of Researcher named "Jane Smith"**"""
+
+g.add((ns.JaneSmith, RDF.type, ns.Reseachers))
```
Comment (dgarijo) [inline Assignment4/NataliaBagnoli-AY3691355/task06.py:48]: You have not create a Researchers class.

**Review by dgarijo — CHANGES_REQUESTED** [review]

File: Assignment4/NataliaBagnoli-AY3691355/task06.py (hunk 1 above)
Comment (NataliaBagnoli) [reply Assignment4/NataliaBagnoli-AY3691355/task06.py]: Yes, sure. Just a typo. Thank you for the correction!

**Pull request merged**